crates/sui-core/src/unit_tests/authority_tests.rs

```diff
@@ -4016,6 +4031,243 @@ async fn test_iter_live_object_set() {
     );
 }
 
+#[tokio::test]
+async fn test_clever_abort_error() {
```

Comment:
I think this test would be more suited to a snapshot/insta test. Otherwise I could see changing this in the future being quite annoying. 

I'm not sure if we already are using insta for some testing in sui-core already, but if not I think this is a good time for us to start using it for this test :)